tests/issues/issue-1022.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+return [
+    'issue-1022' => [
+        'ua' => 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.75 Safari/537.36 Google Favicon',
+        'properties' => [
+            'Comment' => 'Google Favicon',
+            'Browser' => 'Google Favicon',
+            'Browser_Type' => 'Bot/Crawler',
+            'Browser_Bits' => '64',
+            'Browser_Maker' => 'Google Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '0.0',
+            'MajorVer' => '0',
+            'MinorVer' => '0',
+            'Platform' => 'Linux',
+            'Platform_Version' => 'unknown',
+            'Platform_Description' => 'Linux',
+            'Platform_Bits' => '64',
+            'Platform_Maker' => 'Linux Foundation',
+            'Alpha' => false,
+            'Beta' => false,
+            'Win16' => false,
+            'Win32' => false,
+            'Win64' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => false,
+            'BackgroundSounds' => false,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'ActiveXControls' => false,
+            'isMobileDevice' => false,
+            'isTablet' => false,
+            'isSyndicationReader' => false,
+            'Crawler' => true,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '0',
+            'AolVersion' => '0',
+            'Device_Name' => 'Linux Desktop',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Linux Desktop',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'unknown',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'unknown',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-1022 (lite)' => [
```

Comment:
I removed some tests for the lite/standard mode where the browser changed in the result.